cudaextender/samples/host_sample.cu

```diff
@@ -0,0 +1,90 @@
+/*
+* Copyright 2020 NVIDIA CORPORATION.
```

Comment:
Does this have to be a .cu file?

Reply:
Nope. I'll convert it to cpp and add a build for it as soon as I finish refining the device ptr API as requested by Andreas.

Reply:
Removed that file and merged the host pointer sample with the device pointer sample in the same file (sample_cudaextender.cpp). The mode is switched using a flag.